packages/rocketchat-livechat/imports/server/rest/messages.js

```diff
@@ -0,0 +1,64 @@
+import LivechatVisitors from '../../../server/models/LivechatVisitors';
+
+RocketChat.API.v1.addRoute('livechat/messages', {
```

Comment:
You are using `RocketChat.authz.hasPermission`, so you have to set` {authRequired: true} `to the route parameter.